### DISCUSSION TITLE
Python wrapper for greptimedb

### DISCUSSION THREAD

**Discussion by killme2008** [GreptimeTeam/greptimedb#3815] [Ideas]
### What type of enhancement is this?

User experience, Other

### What does the enhancement do?

I think we can wrap greptimedb in python,then the user can just install it by pip:

```sh
pip install greptimedb
```

And access it in python:

```python
import greptimedb

data_home = "/tmp/greptimedb"
db = greptimedb.connect(data_home)

results = db.execute("select * from monitor")
....
```

**Comment by tisonkun** [comment]
Move to Ideas until we have some concrete proposal to proceed.

**Comment by waynexia** [comment]
what's the target use case for this wrapper? An embedded in-process data storage / query engine, Or a Python SDK?

**Reply by killme2008** [reply]
> what's the target use case for this wrapper? An embedded in-process data storage / query engine, Or a Python SDK?

Embedded in-process data storage, just like sqlite.
